website/client/store/actions/auth.js

```diff
@@ -68,5 +68,5 @@ export async function socialAuth (store, params) {
 export function logout () {
   localStorage.removeItem(LOCALSTORAGE_AUTH_KEY);
   localStorage.removeItem(LOCALSTORAGE_SOCIAL_AUTH_KEY);
-  window.location.href = '/logout';
+  axios.get('/logoutServer').then(() => window.location.href = '/');
```

Comment:
Sorry, one more pick :P We use async/await instead of `.then` so you will want to change this function to `async`
Then 
```
await axios.get('/logoutServer');
window.location.href = '/';
```